lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -7,8 +7,7 @@ const ChangeTypes = require('../../ChangeTypes');
 const Negative = require('../../Negative');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
-
-const search = TreeSearch.postOrder(distribute);
+const search = TreeSearch.preOrder(distribute);
```

Comment:
That makes sense -- just seems like a big change and I'd be nervous that it might break something else. But all the tests are passing, so ... probably fine!